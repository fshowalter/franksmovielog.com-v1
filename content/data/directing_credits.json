[{"movie_imdb_id": "tt0019684", "name": "Irving Cummings", "sequence": 0, "person_imdb_id": "nm0191899"}, {"movie_imdb_id": "tt0019699", "name": "John Ford", "sequence": 0, "person_imdb_id": "nm0000406"}, {"movie_imdb_id": "tt0019702", "name": "Alfred Hitchcock", "sequence": 0, "person_imdb_id": "nm0000033"}, {"movie_imdb_id": "tt0019745", "name": "Frank Tuttle", "sequence": 0, "person_imdb_id": "nm0878338"}, {"movie_imdb_id": "tt0019745", "name": "Malcolm St. Clair", "sequence": 1, "person_imdb_id": "nm0820461"}, {"movie_imdb_id": "tt0478970", "name": "Peyton Reed", "sequence": 0, "person_imdb_id": "nm0715636"}, {"movie_imdb_id": "tt2395427", "name": "Joss Whedon", "sequence": 0, "person_imdb_id": "nm0923736"}]