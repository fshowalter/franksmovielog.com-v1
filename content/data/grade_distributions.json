[
{
"grade": "A+",
"gradeValue": 13,
"reviewCount": 7
},
{
"grade": "A",
"gradeValue": 12,
"reviewCount": 10
},
{
"grade": "A-",
"gradeValue": 11,
"reviewCount": 14
},
{
"grade": "B+",
"gradeValue": 10,
"reviewCount": 35
},
{
"grade": "B",
"gradeValue": 9,
"reviewCount": 42
},
{
"grade": "B-",
"gradeValue": 8,
"reviewCount": 55
},
{
"grade": "C+",
"gradeValue": 7,
"reviewCount": 80
},
{
"grade": "C",
"gradeValue": 6,
"reviewCount": 54
},
{
"grade": "C-",
"gradeValue": 5,
"reviewCount": 55
},
{
"grade": "D+",
"gradeValue": 4,
"reviewCount": 101
},
{
"grade": "D",
"gradeValue": 3,
"reviewCount": 45
},
{
"grade": "D-",
"gradeValue": 2,
"reviewCount": 16
},
{
"grade": "F",
"gradeValue": 1,
"reviewCount": 35
}
]