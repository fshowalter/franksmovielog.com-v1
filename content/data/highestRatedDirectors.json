[{"full_name": "Malcolm St. Clair", "rating": 2.708671723565527}, {"full_name": "Frank Tuttle", "rating": 2.708671723565527}, {"full_name": "Alfred Hitchcock", "rating": 2.208671723565527}, {"full_name": "Peyton Reed", "rating": 2.208671723565527}, {"full_name": "Joss Whedon", "rating": 2.208671723565527}, {"full_name": "John Ford", "rating": 1.7086717235655269}, {"full_name": "Irving Cummings", "rating": 1.7086717235655269}]