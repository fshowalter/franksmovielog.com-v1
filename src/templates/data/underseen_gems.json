[
  {
    "imdbId": "tt13872248",
    "title": "The Sadness",
    "year": 2021,
    "releaseDate": "2021-01-22",
    "slug": "the-sadness-2021",
    "sortTitle": "Sadness (2021)",
    "grade": "B+",
    "gradeValue": 10,
    "genres": ["Horror"]
  },
  {
    "imdbId": "tt11060882",
    "title": "Batman: The Dark Knight Returns",
    "year": 2013,
    "releaseDate": "2013-10-08",
    "slug": "batman-the-dark-knight-returns-2013",
    "sortTitle": "Batman: The Dark Knight Returns (2013)",
    "grade": "B",
    "gradeValue": 9,
    "genres": ["Action", "Adventure", "Animation", "Crime", "Drama"]
  },
  {
    "imdbId": "tt1489167",
    "title": "Down Terrace",
    "year": 2009,
    "releaseDate": "2009-09-01",
    "slug": "down-terrace-2009",
    "sortTitle": "Down Terrace (2009)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Crime", "Drama"]
  },
  {
    "imdbId": "tt0096920",
    "title": "Beyond the Door III",
    "year": 1989,
    "releaseDate": "1989-10-01",
    "slug": "beyond-the-door-iii-1989",
    "sortTitle": "Beyond the Door III (1989)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Horror"]
  },
  {
    "imdbId": "tt0093912",
    "title": "Salvation!",
    "year": 1987,
    "releaseDate": "1987-05-31",
    "slug": "salvation-have-you-said-your-prayers-today-1987",
    "sortTitle": "Salvation!: Have You Said Your Prayers Today? (1987)",
    "grade": "B",
    "gradeValue": 9,
    "genres": ["Comedy"]
  },
  {
    "imdbId": "tt0089789",
    "title": "The Peanut Butter Solution",
    "year": 1985,
    "releaseDate": "1985-12-04",
    "slug": "the-peanut-butter-solution-1985",
    "sortTitle": "Peanut Butter Solution (1985)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Adventure", "Comedy", "Family", "Fantasy", "Horror"]
  },
  {
    "imdbId": "tt0084924",
    "title": "Duel to the Death",
    "year": 1983,
    "releaseDate": "1983-01-13",
    "slug": "duel-to-the-death-1983",
    "sortTitle": "Duel to the Death (1983)",
    "grade": "A-",
    "gradeValue": 11,
    "genres": ["Action", "Drama"]
  },
  {
    "imdbId": "tt0078797",
    "title": "Arrebato",
    "year": 1979,
    "releaseDate": "1979-03-02",
    "slug": "arrebato-1979",
    "sortTitle": "Arrebato (1979)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Drama", "Fantasy", "Horror", "Mystery"]
  },
  {
    "imdbId": "tt0074510",
    "title": "Scalpel",
    "year": 1977,
    "releaseDate": "1977-01-01",
    "slug": "scalpel-1977",
    "sortTitle": "Scalpel (1977)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Drama", "Horror", "Thriller"]
  },
  {
    "imdbId": "tt0188527",
    "title": "Island of Death",
    "year": 1976,
    "releaseDate": "1976-01-01",
    "slug": "island-of-death-1976",
    "sortTitle": "Island of Death (1976)",
    "grade": "B",
    "gradeValue": 9,
    "genres": ["Horror", "Romance"]
  },
  {
    "imdbId": "tt0073836",
    "title": "Last Stop on the Night Train",
    "year": 1975,
    "releaseDate": "1975-04-08",
    "slug": "last-stop-on-the-night-train-1975",
    "sortTitle": "Last Stop on the Night Train (1975)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Horror", "Thriller"]
  },
  {
    "imdbId": "tt0070794",
    "title": "The Thief Who Came to Dinner",
    "year": 1973,
    "releaseDate": "1973-03-01",
    "slug": "the-thief-who-came-to-dinner-1973",
    "sortTitle": "Thief Who Came to Dinner (1973)",
    "grade": "B+",
    "gradeValue": 10,
    "genres": ["Comedy", "Romance"]
  },
  {
    "imdbId": "tt0067009",
    "title": "One-Armed Boxer",
    "year": 1972,
    "releaseDate": "1972-08-02",
    "slug": "one-armed-boxer-1972",
    "sortTitle": "One-Armed Boxer (1972)",
    "grade": "B+",
    "gradeValue": 10,
    "genres": ["Action", "Drama"]
  },
  {
    "imdbId": "tt0068306",
    "title": "Bone",
    "year": 1972,
    "releaseDate": "1972-07-01",
    "slug": "bone-1972",
    "sortTitle": "Bone (1972)",
    "grade": "B",
    "gradeValue": 9,
    "genres": ["Comedy", "Drama", "Thriller"]
  },
  {
    "imdbId": "tt0070800",
    "title": "Five Fingers of Death",
    "year": 1972,
    "releaseDate": "1972-05-26",
    "slug": "five-fingers-of-death-1972",
    "sortTitle": "Five Fingers of Death (1972)",
    "grade": "B",
    "gradeValue": 9,
    "genres": ["Action", "Drama", "Romance"]
  },
  {
    "imdbId": "tt0068675",
    "title": "Hannie Caulder",
    "year": 1971,
    "releaseDate": "1971-11-08",
    "slug": "hannie-caulder-1971",
    "sortTitle": "Hannie Caulder (1971)",
    "grade": "B",
    "gradeValue": 9,
    "genres": ["Crime", "Drama", "Western"]
  },
  {
    "imdbId": "tt0064073",
    "title": "The Conspiracy of Torture",
    "year": 1969,
    "releaseDate": "1969-11-14",
    "slug": "the-conspiracy-of-torture-1969",
    "sortTitle": "Conspiracy of Torture (1969)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Drama", "History"]
  },
  {
    "imdbId": "tt0065148",
    "title": "One on Top of the Other",
    "year": 1969,
    "releaseDate": "1969-08-15",
    "slug": "one-on-top-of-the-other-1969",
    "sortTitle": "One on Top of the Other (1969)",
    "grade": "B",
    "gradeValue": 9,
    "genres": ["Mystery", "Thriller"]
  },
  {
    "imdbId": "tt0163114",
    "title": "Murder \u00e0 la Mod",
    "year": 1968,
    "releaseDate": "1968-05-01",
    "slug": "murder-a-la-mod-1968",
    "sortTitle": "Murder \u00e0 la Mod (1968)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Comedy", "Crime", "Mystery", "Thriller"]
  },
  {
    "imdbId": "tt0059358",
    "title": "King Rat",
    "year": 1965,
    "releaseDate": "1965-10-27",
    "slug": "king-rat-1965",
    "sortTitle": "King Rat (1965)",
    "grade": "B",
    "gradeValue": 9,
    "genres": ["Drama", "War"]
  },
  {
    "imdbId": "tt0051444",
    "title": "The Camp on Blood Island",
    "year": 1958,
    "releaseDate": "1958-04-15",
    "slug": "the-camp-on-blood-island-1958",
    "sortTitle": "Camp on Blood Island (1958)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Action", "Drama", "History", "War"]
  },
  {
    "imdbId": "tt0052367",
    "title": "Violent Playground",
    "year": 1958,
    "releaseDate": "1958-01-14",
    "slug": "violent-playground-1958",
    "sortTitle": "Violent Playground (1958)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Crime", "Drama"]
  },
  {
    "imdbId": "tt0038468",
    "title": "It Rains on Our Love",
    "year": 1946,
    "releaseDate": "1946-11-09",
    "slug": "it-rains-on-our-love-1946",
    "sortTitle": "It Rains on Our Love (1946)",
    "grade": "B",
    "gradeValue": 9,
    "genres": ["Drama", "Romance"]
  },
  {
    "imdbId": "tt0038675",
    "title": "Crisis",
    "year": 1946,
    "releaseDate": "1946-02-25",
    "slug": "crisis-1946",
    "sortTitle": "Crisis (1946)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Drama", "Romance"]
  },
  {
    "imdbId": "tt0037614",
    "title": "The Corn Is Green",
    "year": 1945,
    "releaseDate": "1945-03-29",
    "slug": "the-corn-is-green-1945",
    "sortTitle": "Corn Is Green (1945)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Drama"]
  },
  {
    "imdbId": "tt0022321",
    "title": "The Road to Singapore",
    "year": 1931,
    "releaseDate": "1931-10-10",
    "slug": "the-road-to-singapore-1931",
    "sortTitle": "Road to Singapore (1931)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Drama", "Romance"]
  },
  {
    "imdbId": "tt0021508",
    "title": "Up the River",
    "year": 1930,
    "releaseDate": "1930-10-10",
    "slug": "up-the-river-1930",
    "sortTitle": "Up the River (1930)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Comedy", "Crime", "Drama"]
  },
  {
    "imdbId": "tt0020691",
    "title": "The Big Trail",
    "year": 1930,
    "releaseDate": "1930-10-02",
    "slug": "the-big-trail-1930",
    "sortTitle": "Big Trail (1930)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Adventure", "Romance", "Western"]
  },
  {
    "imdbId": "tt0020815",
    "title": "The Dawn Patrol",
    "year": 1930,
    "releaseDate": "1930-07-10",
    "slug": "the-dawn-patrol-1930",
    "sortTitle": "Dawn Patrol (1930)",
    "grade": "B-",
    "gradeValue": 8,
    "genres": ["Action", "Drama", "War"]
  }
]
