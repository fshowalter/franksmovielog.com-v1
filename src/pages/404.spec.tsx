// eslint-disable-next-line import/no-extraneous-dependencies
import { render } from "@testing-library/react";
import NotFoundPage, { Head } from "./404";
import data from "./404.fixtures";

describe("/404", () => {
  it("renders", () => {
    const { asFragment } = render(<NotFoundPage data={data} />);

    expect(asFragment()).toMatchSnapshot();
  });

  it("sets page title", () => {
    render(<Head />);

    expect(document.title).toEqual("404: Not Found");
  });
});
