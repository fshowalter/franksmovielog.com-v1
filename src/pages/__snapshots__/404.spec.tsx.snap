// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/404 renders 1`] = `
<DocumentFragment>
  <a
    class="Box_screenReaderOnlyStyle__1isqo402"
    href="#content"
  >
    Skip to content
  </a>
  <div
    class="Layout_bandStickyStyle__1965ugi1 sprinkles_backgroundImage_ripNotComingSoon__8lcvdk3r sprinkles_minHeight_16_default__8lcvdk8e sprinkles_width_full_default__8lcvdkh0"
  />
  <div
    class="Layout_pageCanvasStyle__1965ugi0 sprinkles_backgroundColor_default__8lcvdk2 sprinkles_display_flex_default__8lcvdk50 sprinkles_flexDirection_column_default__8lcvdk5a"
  >
    <header
      class="Layout_headerLayoutStyle__1965ugi2 sprinkles_paddingLeft_gutter_default__8lcvdkco sprinkles_paddingRight_gutter_default__8lcvdkdw sprinkles_paddingTop_32_default__8lcvdke6 sprinkles_paddingBottom_32_default__8lcvdkba sprinkles_display_flex_default__8lcvdk50 sprinkles_rowGap_24__8lcvdk22 sprinkles_alignItems_center__8lcvdk1v sprinkles_boxShadow_borderBottom__8lcvdk3o sprinkles_backgroundColor_default__8lcvdk2"
    >
      <div
        class="sprinkles_display_flex_default__8lcvdk50 sprinkles_flexDirection_column_default__8lcvdk5a sprinkles_alignItems_inherit__8lcvdk1w sprinkles_justifyItems_inherit__8lcvdk1u"
      >
        <h1
          class="sprinkles_whiteSpace_nowrap__8lcvdk1n sprinkles_fontWeight_normal__8lcvdk2b sprinkles_fontSize_logo__8lcvdk38"
        >
          <a
            class="sprinkles_color_default__8lcvdkv sprinkles_textDecoration_none__8lcvdk3n"
            href="/"
          >
            Frank's Movie Log
          </a>
        </h1>
        <p
          class="Header_taglineStyle__1tt5yzz0 sprinkles_color_muted__8lcvdkw sprinkles_fontStyle_italic__8lcvdk1s sprinkles_lineHeight_16__8lcvdk3e sprinkles_width_full_default__8lcvdkh0"
        >
          My life at the movies.
        </p>
      </div>
      <nav>
        <ul
          class="Nav_listVariants_header__mkdjqm7 Nav_listBaseStyles__mkdjqm2"
        >
          <li
            class="Nav_listItemVariants_header__mkdjqm9 Nav_listItemBaseStyles__mkdjqm1"
          >
            <a
              class="Nav_linkVariants_header__mkdjqm5 Nav_linkBaseStyles__mkdjqm0"
              href="/"
            >
              Home
            </a>
          </li>
          <li
            class="Nav_listItemVariants_header__mkdjqm9 Nav_listItemBaseStyles__mkdjqm1"
          >
            <a
              class="Nav_linkVariants_header__mkdjqm5 Nav_linkBaseStyles__mkdjqm0"
              href="/about/"
            >
              About
            </a>
          </li>
          <li
            class="Nav_listItemVariants_header__mkdjqm9 Nav_listItemBaseStyles__mkdjqm1"
          >
            <a
              class="Nav_linkVariants_header__mkdjqm5 Nav_linkBaseStyles__mkdjqm0"
              href="/how-i-grade/"
            >
              How I Grade
            </a>
          </li>
          <li
            class="Nav_listItemVariants_header__mkdjqm9 Nav_listItemBaseStyles__mkdjqm1"
          >
            <a
              class="Nav_linkVariants_header__mkdjqm5 Nav_linkBaseStyles__mkdjqm0"
              href="/reviews/"
            >
              Reviews
            </a>
          </li>
          <li
            class="Nav_listItemVariants_header__mkdjqm9 Nav_listItemBaseStyles__mkdjqm1"
          >
            <a
              class="Nav_linkVariants_header__mkdjqm5 Nav_linkBaseStyles__mkdjqm0"
              href="/stats/"
            >
              Stats
            </a>
          </li>
          <li
            class="Nav_listItemVariants_header__mkdjqm9 Nav_listItemBaseStyles__mkdjqm1"
          >
            <a
              class="Nav_linkVariants_header__mkdjqm5 Nav_linkBaseStyles__mkdjqm0"
              href="/watchlist/"
            >
              Watchlist
            </a>
          </li>
        </ul>
      </nav>
      <form
        accept-charset="UTF-8"
        action="https://www.google.com/search"
        class="SearchForm_formStyle__b35bo70 sprinkles_maxWidth_prose_default__8lcvdk7w"
        method="get"
        role="search"
      >
        <label
          class="sprinkles_display_flex_default__8lcvdk50 sprinkles_overflow_hidden__8lcvdk1o"
          for="search"
        >
          <div
            class="ScreenReaderOnly_srOnlyCss__rm0eez0"
          >
            Search
          </div>
          <input
            class="SearchForm_searchInputStyle__b35bo71 sprinkles_backgroundColor_subtle__8lcvdk4 sprinkles_color_default__8lcvdkv sprinkles_fontSize_normal__8lcvdk2p sprinkles_fontWeight_light__8lcvdk2a sprinkles_lineHeight_24__8lcvdk3f sprinkles_flexGrow_1__8lcvdk1q"
            id="search"
            name="q"
            placeholder="Search..."
            type="text"
          />
          <input
            name="q"
            type="hidden"
            value="site:www.franksmovielog.com"
          />
          <button
            aria-label="Search"
            class="SearchForm_submitButtonStyle__b35bo72 sprinkles_backgroundColor_subtle__8lcvdk4"
            type="submit"
            value="Search"
          >
            <svg
              class="SearchForm_iconStyle__b35bo73 SvgIcon_iconStyle__agywp20"
              height="1em"
              viewBox="0 0 16 16"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M10.442 10.442a1 1 0 011.415 0l3.85 3.85a1 1 0 01-1.414 1.415l-3.85-3.85a1 1 0 010-1.415z"
                fill-rule="evenodd"
              />
              <path
                d="M6.5 12a5.5 5.5 0 100-11 5.5 5.5 0 000 11zM13 6.5a6.5 6.5 0 11-13 0 6.5 6.5 0 0113 0z"
                fill-rule="evenodd"
              />
            </svg>
          </button>
        </label>
      </form>
    </header>
    <div
      class="sprinkles_flexGrow_1__8lcvdk1q"
    >
      <main>
        <article
          class="sprinkles_display_flex_default__8lcvdk50 sprinkles_flexDirection_column_default__8lcvdk5a sprinkles_alignItems_center__8lcvdk1v"
        >
          <h1
            class="sprinkles_paddingLeft_gutter_default__8lcvdkco sprinkles_paddingRight_gutter_default__8lcvdkdw sprinkles_paddingTop_24_default__8lcvdke4 sprinkles_paddingTop_32_desktop__8lcvdke7 sprinkles_paddingBottom_24_default__8lcvdkb8 sprinkles_paddingBottom_32_desktop__8lcvdkbb sprinkles_fontSize_pageTitle__8lcvdk3a sprinkles_textAlign_center__8lcvdk40"
          >
            404 Not Found
          </h1>
          <div
            class="gatsby-image-wrapper gatsby-image-wrapper-constrained"
            data-gatsby-image-wrapper=""
            style="position: relative; overflow: hidden; display: inline-block; vertical-align: top;"
          >
            <div
              style="max-width: 1000px; display: block;"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;charset=utf-8,%3Csvg height='563' width='1000' xmlns='http://www.w3.org/2000/svg' version='1.1'%3E%3C/svg%3E"
                style="max-width: 100%; display: block; position: static;"
              />
            </div>
            <img
              alt=""
              aria-hidden="true"
              data-placeholder-image=""
              decoding="async"
              src="data:image/svg+xml,%3csvg%20xmlns='http://www.w3.org/2000/svg'%20width='1000'%20height='563'%20viewBox='0%200%201000%20563'%20preserveAspectRatio='none'%3e%3cpath%20d='M0%20282v281h1001V0H0v282m483%20100c-10%202-12%206-19%2040-3%2015-3%2020-3%2026v51h3l4-3c4%200%207-2%208-4%200-2%201-2%204%202%204%204%205%205%2011%205%204%201%206%201%208%204l3%202c1%200%205%205%205%207%200%203%208%209%209%206%201-1%201%200%201%202%200%204%201%204%207%201l5-2v-11a202%20202%200%2000-4-33%20481%20481%200%2000-8-51l-1-7-3-10c-2-7-3-10-8-19-4-7-7-8-22-6m162%205l-4%203-10%201c-7-1-9%200-10%201s-7%202-9%201l-5%201-7%201-6%202c-2%202-4%203-6%202-3%200-3%200-3%205%200%204%200%204-2%204l-2%201c0%202-3%202-5%201h-4l2%202v5c-1%201-4%200-7-3h-1v2l1%207c1%209%205%2016%2010%2016%202%201%204%205%204%208%200%202%200%203%202%203l4%201%204%202%204%202c0%203%201%203%204%204l5%202%204%203%204%203c4%205%207%206%208%203%200-2%201-2%203%200%202%201%202%201%200%203v2h3c0-3%204-2%205%200%202%203%208%204%208%201%200-2%203-2%206%201h4c2-1%203-1%2011%203a195%20195%200%200011%206c1%202%207%205%2011%206%202%200%204%202%205%203s2%202%203%201l2%201h2c1-2%202-2%203-1l6%201c5%200%205%201%205%203%201%205%207%205%208%200l1-1c5%200%2010%202%2012%203h4c2-1%204-1%204%202s2%203%209%204c6%200%208%200%209%202s3%202%207%202c9%200%2022%201%2027%203l10%202%209%201h21l5%201c0%202%202%203%202%201%201-2%204-2%208%200%203%202%2014%203%2016%202l1-4c-1-1-1-2%202-1%202%200%203%202%201%202v1l5-1h3c2%202%205%203%205%201l2-1c2%201%203%200%204-1%202-2%204-1%205%201s1%202%203%201l7-1h5c0-2%2010-4%2013-3l2-1%205-1%208-1h2l1-2c-1-2-1-3%203-3l4%201h1v-1l2-2c2-1%202-1%200-3l-4-3c-1-1-1-1%201-1l4%201h4c2-1%202-1%204%201%201%202%202%202%203%201%202%200%202-1%201-1-2-2-1-3%203-3h5l-3-1-4-1-2-5c-3-3-3-4-1-5%201-2%201-2-2-3l-4-2-7-5c-5-3-6-5-6-6v-3l-3-4-3-2-5-2-4-2-4-2-3-3c-3%200-4%200-4-2s-1-3-5-4l-13-6c-4-2-8-4-11-4l-3-2-2-2-6-2-7-3-4-1-4-2a189%20189%200%2001-32-12l-7-2-9-2-6-2h-2l-12-3-3-1-17-4a244%20244%200%2001-22-4l-17-4-9-1-11-1-12-1-12-1h-3l-11%201-11%201-4-1c-3-2-4-2-10%200m-337%201h-2l-12%201c-3%202-15%203-17%202-1-1-2%200-2%201-1%202-7%203-14%202l-6%202-10%203-8%202-6%201-6%201-5%202-3%201c0%201-2%202-10%204-7%201-13%203-15%205l-5%202-7%202-6%202-3%202-2%202-5%202h-5l-2%202-2%202-14%206-4%202-5%202-3%201h-2l-4%202c-2%200-3%201-3%202s-2%202-5%202c-3%201-5%202-6%204s-3%203-4%202l-3%203c-1%203-1%203-3%202h-2l-3%203-3%203c0%201-8%206-10%206s-7%205-7%207l-3%202c-3%200-9%203-9%204h1l2%202c1%203%201%203%203%202l3-1c1%201%200%204-1%204-3-1-4%203-3%206v4h-3l-1%201c-2%202%208%201%2010-1l14-1h6l2-1-2%203c-3%202-2%203%202%202h5c0%201%201%201%202-1%202-1%204-1%204%202%200%202%203%203%207%200l11-2%2011-2%205-1c2%201%202%200%202-1%200-2%200-2%204-2h3c0-2%202-1%203%202s3%204%205%202l3-1%203-1c0-2%201-2%204-1l4-1h11l6-2c4%200%205-2%207-5l4-1%206-1h8l4-1%207%201c2%201%207-2%2011-5%202-2%204-3%207-3s4%200%205-2l3-2h3c1-2%201-2%203-1h4l4-1c2-1%202-1%201-2h-3v-2l2-2%201%201c2%202%207%204%207%202l2-1%203-1%203-2%204-2%205-2c1%200%202%200%208-6l5-3%203-2%203-2%204-2c1-2%203-3%205-3l2-2%201-2%201%202v2c2%201%205%200%206-3%200-2%201-2%204-2s3%200%203-3%200-3%204-3l5-1%203-2%201-2%202-1%204-1%205-10%201-1c2%200%2012-13%2011-14l-4%201c-3%202-4%202-4%201-1-2-1-3%202-5%203-4%202-5-5-5-5%201-6%201-6-1h-2l-4%201-3%201h-7l2-2c2-2%200-4-5-4l-7-1h-3c-1%202-2%202-7%201l-7-2-2%201'%20fill='%23d3d3d3'%20fill-rule='evenodd'/%3e%3c/svg%3e"
              style="height: 100%; left: 0px; position: absolute; top: 0px; width: 100%;"
            />
            <picture>
              <source
                data-srcset="/static/1b59d2a682b5e99753024caedc9d1323/fdfcf/not-found.avif 250w,/static/1b59d2a682b5e99753024caedc9d1323/b400f/not-found.avif 500w,/static/1b59d2a682b5e99753024caedc9d1323/70674/not-found.avif 1000w,/static/1b59d2a682b5e99753024caedc9d1323/31a81/not-found.avif 2000w"
                sizes="(min-width: 1000px) 1000px, 100vw"
                type="image/avif"
              />
              <img
                alt="A lost highway."
                data-gatsby-image-ssr=""
                data-main-image=""
                data-src="/static/1b59d2a682b5e99753024caedc9d1323/f4320/not-found.jpg"
                data-srcset="/static/1b59d2a682b5e99753024caedc9d1323/2af26/not-found.jpg 250w,/static/1b59d2a682b5e99753024caedc9d1323/a5dc9/not-found.jpg 500w,/static/1b59d2a682b5e99753024caedc9d1323/f4320/not-found.jpg 1000w,/static/1b59d2a682b5e99753024caedc9d1323/f2565/not-found.jpg 2000w"
                decoding="async"
                loading="lazy"
                sizes="(min-width: 1000px) 1000px, 100vw"
                style="height: 100%; left: 0px; position: absolute; top: 0px; transform: translateZ(0); transition: opacity 250ms linear; width: 100%; will-change: opacity; opacity: 0;"
              />
            </picture>
            <noscript />
            <script
              type="module"
            >
              const t="undefined"!=typeof HTMLImageElement&&"loading"in HTMLImageElement.prototype;if(t){const t=document.querySelectorAll("img[data-main-image]");for(let e of t){e.dataset.src&&(e.setAttribute("src",e.dataset.src),e.removeAttribute("data-src")),e.dataset.srcset&&(e.setAttribute("srcset",e.dataset.srcset),e.removeAttribute("data-srcset"));const t=e.parentNode.querySelectorAll("source[data-srcset]");for(let e of t)e.setAttribute("srcset",e.dataset.srcset),e.removeAttribute("data-srcset");e.complete&&(e.style.opacity=1,e.parentNode.parentNode.querySelector("[data-placeholder-image]").style.opacity=0)}}
            </script>
          </div>
          <span
            class="sprinkles_width_px_default__8lcvdkgy sprinkles_height_64_default__8lcvdk5s sprinkles_minWidth_px_default__8lcvdkay sprinkles_minHeight_64_default__8lcvdk8o sprinkles_display_block_default__8lcvdk4w"
          />
          <div
            class="RenderedMarkdown_renderedMarkdownStyle__1vb8ogj0 sprinkles_paddingLeft_gutter_default__8lcvdkco sprinkles_paddingRight_gutter_default__8lcvdkdw sprinkles_fontWeight_light__8lcvdk2a sprinkles_letterSpacing_0.3__8lcvdk3l sprinkles_fontSize_prose__8lcvdk37 sprinkles_maxWidth_proseWithGutters_default__8lcvdk7y"
          >
            <p>
              This is probably my fault. Are you looking for one of my old reviews? I’m in the process of touching them up
and reposting them. In the mean time, you can find them in the site’s 
              <a
                href="https://github.com/fshowalter/franksmovielog.com/"
              >
                GitHub repo
              </a>
              .
            </p>
            

            <p>
              If you still can’t find what you need, 
              <a
                href="https://github.com/fshowalter/franksmovielog.com/issues/new/choose"
              >
                open an issue on GitHub
              </a>
               and I’ll try to help.
            </p>
            

            <p>
              My apologies, and thanks for stopping by.
            </p>
          </div>
          <span
            class="sprinkles_width_px_default__8lcvdkgy sprinkles_height_128_default__8lcvdk60 sprinkles_minWidth_px_default__8lcvdkay sprinkles_minHeight_128_default__8lcvdk8w sprinkles_display_block_default__8lcvdk4w"
          />
        </article>
      </main>
    </div>
    <footer
      class="sprinkles_paddingLeft_gutter_default__8lcvdkco sprinkles_paddingRight_gutter_default__8lcvdkdw sprinkles_paddingTop_32_default__8lcvdke6 sprinkles_paddingBottom_32_default__8lcvdkba sprinkles_color_inverse__8lcvdkz sprinkles_display_flex_default__8lcvdk50 sprinkles_backgroundImage_ripNotComingSoon__8lcvdk3r sprinkles_rowGap_24__8lcvdk22 sprinkles_alignItems_center__8lcvdk1v sprinkles_flexDirection_column_default__8lcvdk5a"
    >
      <nav>
        <ul
          class="Nav_listVariants_footer__mkdjqm8 Nav_listBaseStyles__mkdjqm2"
        >
          <li
            class="Nav_listItemVariants_footer__mkdjqma Nav_listItemBaseStyles__mkdjqm1"
          >
            <a
              class="Nav_linkVariants_footer__mkdjqm6 Nav_linkBaseStyles__mkdjqm0"
              href="/"
            >
              Home
            </a>
          </li>
          <li
            class="Nav_listItemVariants_footer__mkdjqma Nav_listItemBaseStyles__mkdjqm1"
          >
            <a
              class="Nav_linkVariants_footer__mkdjqm6 Nav_linkBaseStyles__mkdjqm0"
              href="/about/"
            >
              About
            </a>
          </li>
          <li
            class="Nav_listItemVariants_footer__mkdjqma Nav_listItemBaseStyles__mkdjqm1"
          >
            <a
              class="Nav_linkVariants_footer__mkdjqm6 Nav_linkBaseStyles__mkdjqm0"
              href="/how-i-grade/"
            >
              How I Grade
            </a>
          </li>
          <li
            class="Nav_listItemVariants_footer__mkdjqma Nav_listItemBaseStyles__mkdjqm1"
          >
            <a
              class="Nav_linkVariants_footer__mkdjqm6 Nav_linkBaseStyles__mkdjqm0"
              href="/reviews/"
            >
              Reviews
            </a>
          </li>
          <li
            class="Nav_listItemVariants_footer__mkdjqma Nav_listItemBaseStyles__mkdjqm1"
          >
            <a
              class="Nav_linkVariants_footer__mkdjqm6 Nav_linkBaseStyles__mkdjqm0"
              href="/stats/"
            >
              Stats
            </a>
          </li>
          <li
            class="Nav_listItemVariants_footer__mkdjqma Nav_listItemBaseStyles__mkdjqm1"
          >
            <a
              class="Nav_linkVariants_footer__mkdjqm6 Nav_linkBaseStyles__mkdjqm0"
              href="/watchlist/"
            >
              Watchlist
            </a>
          </li>
        </ul>
      </nav>
      <p
        class="sprinkles_fontWeight_light__8lcvdk2a sprinkles_fontSize_small__8lcvdk2o sprinkles_lineHeight_16__8lcvdk3e"
      >
        All stills used in accordance with the 
        <a
          class="sprinkles_color_inherit__8lcvdk10 sprinkles_textDecoration_none__8lcvdk3n"
          href="http://www.copyright.gov/title17/92chap1.html#107"
        >
          Fair Use Law.
        </a>
      </p>
      <a
        class="Box_screenReaderOnlyStyle__1isqo402"
        href="#top"
      >
        To the top ↑
      </a>
    </footer>
  </div>
</DocumentFragment>
`;
